CodingStyle

```diff
@@ -28,12 +28,12 @@ by section.
 
     Google uses CamelCaps for all type names.  We use two naming schemes:
 
-    - for naked structs (simple data containers), lower case with _d
-      suffix ('d' for data).  Not _t, because that means typdef.
+    - for naked structs (simple data containers), lower case with _t.
+      Yes, _t also means typedef.  It's perhaps not ideal.
 
-        struct my_type_d {
+        struct my_type_t {
           int a, b;
-          my_type_d() : a(0), b(0) {}
+          my_type_t() : a(0), b(0) {}
```

Comment:
I'm fine with this, but it might be worth updating the example while we're here, ie.:
struct my_type_t {
 int a = 0, b = 0;
};